### DISCUSSION TITLE
能不能整个强制开启8k选项和HDR和杜比的插件

### DISCUSSION THREAD

**Discussion by lrs95** [the1812/Bilibili-Evolved#3349] [功能建议 / Ideas]
不知道能不能实现

**Comment by Geeyun-JY3** [comment]
那也得片源是才行啊……

**Reply by lrs95** [reply]
哔站有支持的片源，但要Safari最新版浏览器才支持8K，HDR也只有手机客户端和旧版edge浏览器支持，杜比目前只有在手机上开启过。虽然可以通过更改浏览器UA的方法强开8k，但支持不是很兼容，而且不方便，不能同时开hdr和8k，要来回切换

**Reply by Geeyun-JY3** [reply]
之前有些相关的下载讨论：#2589、#1484、#2596、#1946

**Reply by lrs95** [reply]
恩，主要是想在线看，下载倒无所谓
